### DISCUSSION TITLE
Display which user(s) have requested an installation of Copilot Business

### DISCUSSION THREAD

**Discussion by NotMyFault** [community/community#78179] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Occasionally I receive emails like

![Screenshot 2023-12-07 at 18 19 20](https://github.com/community/community/assets/13383509/135e08cc-4ed0-488a-bd98-ca828d38fd23)

outlining that users in my organization are interested in copilot business. I administer organizations with several thousand members, making it impossible to see who requested such a feature.
It would be handy to see a compelling list of users who have requested an installation of copilot business in the "Request from members" tab on GitHub, given I can't reach out to everyone individually.

**Comment by netflash** [comment]
It looks like this didn't get any attention from GitHub, doesn't it?
